packages/safe-ds-vscode/src/extension/mainClient.ts

```diff
@@ -178,8 +180,59 @@ const registerVSCodeCommands = function (context: vscode.ExtensionContext) {
                         vscode.window.showErrorMessage('No .sdspipe file selected!');
                         return;
                     }
+
+                    // Getting of pipeline name
+                    const document = await getPipelineDocument(editor.document.uri);
+                    if (!document) {
+                        vscode.window.showErrorMessage('Internal error!');
```

Comment:
We should not "scream" at the user in error messages. Rather end with a full stop than with an exclamation point.